content/backend/graphql-scala/3-handling_arguments.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Handling Arguments
+pageTitle: "Handling arguments which are provided in query"
+description: "In this chapter, at first you will learn how to use arguments."
+question: "Does GraphQL needs HTTP Server?"
+answers: ["Yes. It needs HTTP server.", "Yes, it needs HTTP server but some of features can be used without that", "No, but it strictly recommended to use. Without HTTP layer, GraphQL is losing some of its features.","No, GraphQL is specification is far away from tranportation protocol. You can use HTTP, Websockets, sockets or even use it internally in you application." ]
+correctAnswer: 3
+---
+
+
+### Arguments
+
+Let's assume, we want to fetch links for only one link (giving ID of it) or few links. We will need to manage parsing of arguments.
```

Comment:
`fetch links for only one link` too many links :) ->`fetch selected links using their ids` ? 
you can drop `manage`